src/cpp/shared/game/game.cpp

```diff
@@ -214,11 +214,11 @@ void game::PerMonth() {
 }
 
 void game::ResetIronfistGameState() {
```

Comment:
Revert this as well. I recommend inspecting your git status before making any commits so you don't accidentally commit whitespace changes like this.